### ISSUE TITLE
index 50 is out of bounds for axis 0 with size 50

### ISSUE THREAD

**Issue by tracob-github** [lucasmansilla/ACRN_Chest_X-ray_IA#16]
Thanks for sharing this excellent code.

When I run test_aenet.py, an error occurred.
![image](https://user-images.githubusercontent.com/59277274/134470686-f7f74c26-3a94-4161-ac55-8738122c6d51.png)
After debugging, I found that the size of test_lbs and z in models.py are inconsistent. The size of test_lbs is (50, 256, 256, 1), but the size of z is (800, 64, 64, 1). As show in two pictures.
![1632383057](https://user-images.githubusercontent.com/59277274/134471158-151ae87b-04f8-4e67-b8f4-372de45d1313.png)
![1632383057(1)](https://user-images.githubusercontent.com/59277274/134471180-c19f632a-9643-4d92-8134-348bd366ce7c.png)

 I would like to ask how to sovle it.

**Comment by lucasmansilla** [comment]
Hi @tracob-github,

AE-Net runs with 64x64 segmentations. What size of segmentations are you using?

**Comment by Westbrone** [comment]
Hello, your code is written very well, but because there is no pre-training model of the autoencoder, when I was training the model, the loss remained at around 0.49 in the end. Is it because of the pre-training model?
